scripts/build-lds.ts

```diff
@@ -0,0 +1,90 @@
+'use strict';
```

Comment:
Not sure it's sensible to move this file. If we just add it to the `.vscodeignore` it shouldn't matter, right? Tehcnically, all of these are scripts...